wrapper/src/main/java/software/amazon/jdbc/dialect/RdsMultiAzDbClusterPgDialect.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package software.amazon.jdbc.dialect;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.List;
+import software.amazon.jdbc.exceptions.ExceptionHandler;
+import software.amazon.jdbc.exceptions.MultiAzDbClusterPgExceptionHandler;
+import software.amazon.jdbc.hostlistprovider.RdsMultiAzDbClusterListProvider;
+import software.amazon.jdbc.util.DriverInfo;
+
+public class RdsMultiAzDbClusterPgDialect extends PgDialect {
+
+  private static MultiAzDbClusterPgExceptionHandler exceptionHandler;
+
+  private static final String TOPOLOGY_QUERY =
+      "SELECT LOWER(id) AS id, LOWER(endpoint) AS endpoint, port FROM "
+      + "rds_tools.show_topology('aws_jdbc_driver-" + DriverInfo.DRIVER_VERSION + "')";
+
+  private static final String FETCH_WRITER_NODE_QUERY =
+      "SELECT LOWER(multi_az_db_cluster_source_dbi_resource_id) AS multi_az_db_cluster_source_dbi_resource_id FROM "
+      + "rds_tools.multi_az_db_cluster_source_dbi_resource_id()";
+
+  private static final String FETCH_WRITER_NODE_QUERY_HEADER = "multi_az_db_cluster_source_dbi_resource_id";
```

Comment:
QUERY_HEADER seems like the wrong name

Reply:
Will use `FETCH_WRITER_NODE_QUERY_COLUMN_NAME`.